stacker/providers/aws.py

```diff
@@ -5,10 +5,23 @@
 
 from . import exceptions
 from .base import BaseProvider
+from ..util import retry_with_backoff
 
 logger = logging.getLogger(__name__)
 
 
+def retry_cf_throttling(fn, attempts=3, args=None, kwargs=None):
```

Comment:
you could also just take **kwargs instead of having to pass them in as `kwargs=dict`